fenix/app/src/main/java/org/mozilla/fenix/components/appstate/AppStoreReducer.kt

```diff
@@ -100,6 +101,10 @@ internal object AppStoreReducer {
                 else -> state.recentSyncedTabState
             },
         )
+        is AppAction.SelectedTabChanged -> state.copy(
```

Comment:
Let's add some tests for this in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/test/java/org/mozilla/fenix/components/appstate/AppStoreReducerTest.kt